### PR TITLE
Improve Docker build logs.

### PR DESCRIPTION
**Pull request by fa9r**
## Describe changes
I improved the logs when building Docker images (e.g. when using remote orchestrators):
- all external requirements are now explicitly mentioned in `info` logs,
- rephrased log messages related to `.dockerignore`,
- send Docker logs to `debug` instead of `info`.

Screenshot 2022-07-25 at 11 39 18

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/advanced-guide/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Other (add details above)

### CODE REVIEW

**Review by schustmi — APPROVED** [review]

**Comment by schustmi** [comment]
@fa9r Can we merge this quickly (once someone else has approved and CI passes) please so I can base my docker branch (which will rework this entire thing) on top of that to not lose anything

**Comment by fa9r** [comment]
@schustmi merged :)

**Pull request merged**
